Tasks/PowerShellOnTargetMachines2/task.json

```diff
@@ -0,0 +1,233 @@
+{
+    "id": "3B5693D4-5777-4FEE-862A-BD2B7A374C68",
+    "name": "PowerShellOnTargetMachines",
+    "friendlyName": "PowerShell on Target Machines",
+    "description": "Execute PowerShell scripts on remote machine(s) using Invoke-Command cmdlet",
+    "helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?linkid=627414)",
+    "category": "Deploy",
+    "visibility": [
+        "Build",
+        "Release"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 3,
+        "Minor": 1,
+        "Patch": 101
+    },
+    "preview": true,
+    "groups": [
+        {
+            "name": "ScriptOptions",
+            "displayName": "Script options",
+            "isExpanded": true
+        },
+        {
+            "name": "SessionOptions",
+            "displayName": "PSSession options",
+            "isExpanded": true
+        },
+        {
+            "name": "ErrorHandlingOptions",
+            "displayName": "Error handling options",
+            "isExpanded": true
+        },
+        {
+            "name": "advanced",
+            "displayName": "Advanced",
+            "isExpanded": false
+        }
+    ],
+    "releaseNotes": "What's new in Version 3.0: <br/>&nbsp;&nbsp; Use invoke-command to perform remote script execution",
+    "minimumAgentVersion": "2.0.0",
+    "inputs": [
+        {
+            "name": "Machines",
+            "type": "multiLine",
+            "label": "Machines",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "Provide a comma separated list of machine IP addresses or FQDNs along with ports. Port is defaulted based on the selected protocol. <br>Eg: dbserver.fabrikam.com,dbserver_int.fabrikam.com:5986,192.168.12.34:5986 <br>Or provide build or release variables. Eg: $(variableName) <br>If you are using HTTPS, name/IP of machine should match the CN in the certificate."
+        },
+        {
+            "name": "UserName",
+            "type": "string",
+            "label": "Username",
+            "defaultValue": "",
+            "helpMarkDown": "Username for the target machines. The username should be a part of Administrators group or WinRM remote management users group. <br> Eg: Format: Domain\\Admin User, Admin User@Domain, .\\Admin User"
```

Comment:
Since this is not a mandatory input, can you call out in help what is the credential used if no input is given.